x-pack/plugins/security_solution/server/lib/telemetry/helpers.test.ts

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
These tests are great to make sure we don't have any task scheduling scariness, since there were some recent Task Manager  performance scares.